### PR TITLE
Update manage user accounts 

### PR DESCRIPTION
**Pull request by matthew-robinson-ons**
# What and why?
The Manage use accounts and manage account pages need updating to improve usability. This PR separates the edit and delete page functionality from the manage user accounts page along with some updates to the delete user account page and manage account page. 
# How to test?
Go through the edit user account journey, including deleting an account and editing permissions on an exisiting account to ensure all buttons function as expected and redirect correctly. 
# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&projectKey=RAS&view=detail&selectedIssue=RAS-271

### CODE REVIEW

File: response_operations_ui/templates/admin/manage-user-accounts.html [hunk 1]
```diff
@@ -109,7 +109,7 @@ <h1 class="ons-u-fs-l u-mb-xs" name="page-messages-title">Manage user accounts</
                         "value": "Last name",
                     },
                     { 
-                        "value": ""
+                        "value": " "
```
[the remark this answers is not in the document]
Comment (matthew-robinson-ons) [reply response_operations_ui/templates/admin/manage-user-accounts.html:112]: No, will take that out

File: response_operations_ui/templates/admin/manage-user-accounts.html [hunk 2]
```diff
@@ -130,8 +130,8 @@ <h1 class="ons-u-fs-l u-mb-xs" name="page-messages-title">Manage user accounts</
                                 "value": user.lastname,
                             },
                             {
-                                "value": '<a href="' + url_for("admin_bp.get_manage_account_groups", user_id=user.id) + '">Edit or delete user account </a>' if user.show_edit_button else "",
-                                "class": "ons-u-ta-right"
+                                "value": '<a href="' + url_for("admin_bp.get_manage_account_groups", user_id=user.id) + '" > Edit</a>' + '<a style="text-decoration: none"> | </a>' + '<a href="' + url_for("admin_bp.get_delete_uaa_user", user_id=user.id) + '"> Delete </a>' if user.show_edit_button else "",
```
[the remark this answers is not in the document]
Comment (LJBabbage) [reply response_operations_ui/templates/admin/manage-user-accounts.html:133]: hang on is the "|" and hyperlink with style?

[the remark this answers is not in the document]
Comment (matthew-robinson-ons) [reply response_operations_ui/templates/admin/manage-user-accounts.html:133]: Was a pattern i copied from surveys.html line 99:
                     `   "value": '<a href="/surveys/' + survey.shortName.replace(' ', '') + '" name="survey-link-' + survey.shortName + '">' + survey.longName + '</a><a style="text-decoration: none">' + ' | ' + survey.shortName + '</a>',`
                     
Though i did think that inline styling was avoided?

File: tests/views/test_admin.py [hunk 3]
```diff
@@ -133,7 +133,8 @@ def test_manage_user_accounts_success(self, mock_request):
         self.assertIn("Create new user account".encode(), response.data)
         self.assertIn("Andy0".encode(), response.data)
         self.assertIn("Johnson".encode(), response.data)
-        self.assertIn("Edit or delete user account".encode(), response.data)
+        self.assertIn("Edit".encode(), response.data)
```
[the remark this answers is not in the document]
Comment (matthew-robinson-ons) [reply tests/views/test_admin.py:136]: Agreed, i thought the same but couldn't work out a better assert for that test.

File: response_operations_ui/templates/admin/manage-user-accounts.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (matthew-robinson-ons) [reply response_operations_ui/templates/admin/manage-user-accounts.html]: This has been updated

File: response_operations_ui/templates/admin/manage-user-accounts.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (matthew-robinson-ons) [reply response_operations_ui/templates/admin/manage-user-accounts.html]: This code has been updated to remove the <a> tag

File: response_operations_ui/templates/admin/manage-user-accounts.html [hunk 4]
```diff
@@ -130,8 +130,8 @@ <h1 class="ons-u-fs-l u-mb-xs" name="page-messages-title">Manage user accounts</
                                 "value": user.lastname,
                             },
                             {
-                                "value": '<a href="' + url_for("admin_bp.get_manage_account_groups", user_id=user.id) + '">Edit or delete user account </a>' if user.show_edit_button else "",
-                                "class": "ons-u-ta-right"
+                                "value": '<a href="' + url_for("admin_bp.get_manage_account_groups", user_id=user.id) + '" > Edit</a>' + ' | ' + '<a href="' + url_for("admin_bp.get_delete_uaa_user", user_id=user.id) + '"> Delete </a>' if user.show_edit_button else "",
```
Comment (LJBabbage) [inline response_operations_ui/templates/admin/manage-user-accounts.html:133]: Not sure the white space leading space around Edit and Delete does anything, when I change it, it doesn't move (i.e > Edit</a>)

**Review by LJBabbage — APPROVED** [review]
Works as expect, one minor thing, but not worth blocking

**Pull request merged**